spec/features/home_page_spec.rb

```diff
@@ -1,8 +1,8 @@
-require "rails_helper"
+require 'rails_helper'
 
 describe 'Home Page' do
   it 'should have the application name as the page title' do
     visit '/'
-    expect(page).to have_title("SUL Requests")
+    expect(page).to have_title('SUL Requests')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.